lib/activeadmin_addons/engine.rb

```diff
@@ -2,16 +2,7 @@ module ActiveAdminAddons
   module Rails
     class Engine < ::Rails::Engine
       require "select2-rails"
-      require "sass"
-      begin
-        require 'sassc-rails'
-      rescue LoadError
-        begin
-          require 'sass-rails'
-        rescue LoadError
-          raise "Couldn't find 'sass-rails' or 'sassc-rails' gems in your application."
-        end
-      end
+      require 'sassc-rails'
```

Comment:
Shouldn't this two lines be replaced for:
```ruby
require 'sassc'
require 'sassc-rails'
```
And not only `sassc-rails`?

I'm not sure if this change is necessary, but before your changes, the two old gems where required, `sass` and `sass-rails`, so it seems logical to replace them for the two new gems, not only the second one, let me know what you think.

Reply:
Makes sense to me, updated!